makers/senior_engineer.md

```diff
@@ -4,32 +4,44 @@ Senior Engineer
-* Show curiosity to not only learn new things but fully understand how they work 
-* Be highly productive - have a reputation for getting things done quickly and efficiently
-* Be a mentor for other engineers
-* Require very little oversight from manager/team leads
-* Own and complete full projects beginning with identifying and communicating the problems to be solved, getting and incorporating feedback on proposed architectural solutions, and making a final decision as the owner of a project.
-* Deconstruct a problem into an executable action plan for themselves and other engineers - also perform them in a high quality way without issue
+### Job expectations
+#### Results
+*Focus on understanding what you need to ship and why. Clearly communicate how you intend to deliver user value with reasonably accurate estimates.*
+* Build your expertise with multiple areas or domains of the code/product.
+* Show curiosity to not only learn new things but fully understand how they work.
+* Be consistently productive; have a reputation for getting things done efficiently to iteratively delivery value.
+* Be trusted to work independently because you are collaborative by nature and communicate proactively.
+* Deconstruct a problem into an executable action plan for yourselves and other engineers to implement.
 * Consistently tie back a project to a team goal
-* Set and maintain high individual and team expectations
+* Own and complete full projects beginning with identifying and communicating the problems to be solved, getting and incorporating feedback on proposed architectural solutions, and making a final decision as the owner of a project.
+
+#### Relationships
+*Focus on building your reputation within and across teams as a trusted collaborator who provides valuable feedback and is unafraid to make decisions and clearly communicate their rational.*
+
+* Mentor other engineers and assist with onboarding team members into product areas where you have expertise.
 * Actively participate in frequent code/design/architecture reviews
-* Be able to communicate well with all engineers regardless of seniority
-* Generate support for a company/team decision
-* Work across engineering teams (dev, QA, systems) to ensure new code launches smoothly and customer needs are met
-* Monitor the outcomes of team’s actions to learn from mistakes and replicate successes
-* Interview prospective candidates and provide thoughtful evaluations and feedback
+* Be able to communicate well with all colleagues regardless of seniority.
+* Understand the operational impact of your changes and collaborate with the Operations team for the productionizing of your changes as needed.
+* Understand the security impacts of your changes and work closely with the Security team to conduct reviews and mitigate risks.
+* Understand the business value of your projects and partner with your PM and Data engineer to contribute to accurate analytics and measurements.
+
```

Comment:
Maybe something about "Ensure the quality of code meets Mozilla's standards by engaging with the QA team" -- or something with better wording. :)  I think a relationship with QA or some quality assurance plan is important

Reply:
@clouserw I realized that is missing, but I wonder if it should be part of the P2 description.  What do you think?